This PR should fix most of the test cases for the different platforms.

tpersson opened the pull request.

ldeniau commented:
Hi Tobias,

About the change of sign in s-rotation, I propose to raise the question during the meeting on aperture (did you received an invitation, I asked to Alexander to invite you?) as it is a change that breaks the backward compatibility…

Best,
Laurent.

On 15 Feb 2018, at 11:31, tpersson <notifications@github.com<mailto:notifications@github.com>> wrote:

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570

Commit Summary

  *   Changed the ref files for the output for the rotation tests.
  *   A new survey file since the rotations have changed in survey.
  *   Updated some cfg files. It seems like a line was shifted in some cases.
  *   More updates of cfg files. No big changes to pass the tests.

File Changes

  *   M tests/test-aperture-2/test-aperture-2.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-0> (4)
  *   M tests/test-cororbit-4/test-cororbit-4.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-1> (2)
  *   M tests/test-ibs-4/test-ibs-4.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-2> (1)
  *   M tests/test-makethin-2/test-makethin-2.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-3> (1)
  *   M tests/test-makethin-3/test-makethin-3.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-4> (1)
  *   M tests/test-ptc-twiss-2/test-ptc-twiss-2.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-5> (2)
  *   M tests/test-sequence-4/all_survey.ref<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-6> (112)
  *   M tests/test-track-11/test-track-11.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-7> (2)
  *   M tests/test-track-rotations/test-track-rotations.ref<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-8> (3)
  *   M tests/test-xrotation/test-xrotation.ref<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-9> (3)
  *   M tests/test-yrotation/test-yrotation.ref<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-10> (3)

Patch Links:

  *   https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570.patch
  *   https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABKFz92G2fndIHVRRiGu9uYBxXNZcneBks5tVAeCgaJpZM4SGoIw>.

tpersson commented:
Hi Laurent,
Yes I got the invitation to the meeting and I fully agree that it is a good moment to raise the question.

cheers
Tobias
________________________________
From: Laurent Deniau [notifications@github.com]
Sent: 15 February 2018 11:40
To: MethodicalAcceleratorDesign/MAD-X
Cc: Tobias Hakan Bjorn Persson; Author
Subject: Re: [MethodicalAcceleratorDesign/MAD-X] This PR should fix most of the test cases for the different platforms. (#570)

Hi Tobias,

About the change of sign in s-rotation, I propose to raise the question during the meeting on aperture (did you received an invitation, I asked to Alexander to invite you?) as it is a change that breaks the backward compatibility…

Best,
Laurent.

On 15 Feb 2018, at 11:31, tpersson <notifications@github.com<mailto:notifications@github.com>> wrote:

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570

Commit Summary

* Changed the ref files for the output for the rotation tests.
* A new survey file since the rotations have changed in survey.
* Updated some cfg files. It seems like a line was shifted in some cases.
* More updates of cfg files. No big changes to pass the tests.

File Changes

* M tests/test-aperture-2/test-aperture-2.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-0> (4)
* M tests/test-cororbit-4/test-cororbit-4.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-1> (2)
* M tests/test-ibs-4/test-ibs-4.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-2> (1)
* M tests/test-makethin-2/test-makethin-2.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-3> (1)
* M tests/test-makethin-3/test-makethin-3.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-4> (1)
* M tests/test-ptc-twiss-2/test-ptc-twiss-2.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-5> (2)
* M tests/test-sequence-4/all_survey.ref<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-6> (112)
* M tests/test-track-11/test-track-11.cfg<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-7> (2)
* M tests/test-track-rotations/test-track-rotations.ref<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-8> (3)
* M tests/test-xrotation/test-xrotation.ref<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-9> (3)
* M tests/test-yrotation/test-yrotation.ref<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570/files#diff-10> (3)

Patch Links:

* https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570.patch
* https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABKFz92G2fndIHVRRiGu9uYBxXNZcneBks5tVAeCgaJpZM4SGoIw>.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/MethodicalAcceleratorDesign/MAD-X/pull/570#issuecomment-365888867>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFBJZo49p2a4DQOd_b2Yq_0ENFOUMCIMks5tVAmNgaJpZM4SGoIw>.
